ui/components/app/snaps/snap-ui-asset-selector/useSnapAssetDisplay.tsx

```diff
@@ -120,7 +107,10 @@ export const useSnapAssetSelectorData = ({
       icon: asset.image,
       symbol: asset.symbol,
       name: asset.title,
-      balance: formatAssetBalance(asset.primary),
+      balance: formatCurrencyWithMinThreshold(
+        Number(asset.balance ?? 0),
+        currentCurrency,
```

Comment:
Shouldn't we use the asset symbol ?

```suggestion
        asset.symbol,
```

Reply:
The formatters take care of that, hence you'll see in the snapshot diff: 
<img width="91" height="119" alt="image" src="https://github.com/user-attachments/assets/fc33b374-89e7-46d4-8dfb-f395cc4ffa11" />